### DISCUSSION TITLE
Having problem verifing that i am a student and elidigible for student program.

### DISCUSSION THREAD

**Discussion by Sailesh-Singh** [community/community#77818] [GitHub Education]
### Hi everyone,

What i am having a problem is  due to college multiple mail account for verification on of the account got pruned by the college.
when i try verify it  that i am student  with my new college  mail address and a  college documents ,it always prompt me with

``Your application cannot be reviewed until you fix the following:
Your uploaded proof has already been used previously.
``

I  am  having a problem with  verification so what  can be done to  insure that i can claim my  student pack 
![Screenshot from 2023-12-04 11-10-54](https://github.com/community/community/assets/58818777/2bbccc32-a09f-44fd-9358-9e9d13c31555)
.

**Comment by aramshiva** [comment]
This error appears if your college documents were the same as the ones uploaded before.

**Comment by dakshithadissanayaka** [comment]
It sounds like you are having trouble verifying your student status on GitHub due to a previous email account being pruned by your college. This is causing GitHub to reject your verification requests because the proof of enrollment you are providing has already been used with a different account.

To resolve this issue, you can contact GitHub support and explain the situation. They may be able to manually verify your student status and grant you access to the student pack.

Here are the steps to contact GitHub support:

1. Go to the GitHub Education support page: [https://education.github.com/benefits](https://education.github.com/benefits): [https://education.github.com/benefits](https://education.github.com/benefits).
2. Click the **Contact us** button.
3. Select **I have a problem with my GitHub Education account** as the issue category.
4. In the **Description** field, provide a detailed explanation of the issue, including the fact that you are unable to verify your student status due to a previous email account being pruned by your college and that you have already tried verifying with your new college email address and proof of enrollment.
5. Attach the screenshot you provided in your post to the ticket.
6. Click the **Submit** button.

Once you have submitted a support ticket, a GitHub Education representative will review your request and respond as soon as possible.

In the meantime, you can try the following workaround:

1. Create a new GitHub account using your new college email address.
2. Apply for the student pack on your new account.
3. When prompted to provide proof of enrollment, submit a different document than the one you submitted on your previous account.
4. Once your verification is approved, you can link your new account to your main account using the **Manage accounts** page.

This workaround will allow you to claim the student pack on a new account, even though your previous account has been pruned.

I hope this information is helpful. Please let me know if you have any other questions.

**Comment by Sailesh-Singh** [comment]
As I have already mentioned, I am currently enrolled in the _**same program in same college but my education mail is only changed**_. Is anyone here, who can help me out to resolve this issue.

**Comment by Saurav-IIITU** [comment]
I also got this same message while I was uploading the ID Card of my college. Then I uploaded my recent report card and it worked for me as they were asking for recent document. hope this will help you.

**Reply by Sailesh-Singh** [reply]
Thanks @Saurav-IIITU I will give a try to verify using your stated method
